Software/Python/easygopigo.py

```diff
@@ -42,7 +42,7 @@ def _ifMutexRelease(mutex_enabled = False):
     is_line_follower_accessible = True
 except:
     try:
-        sys.path.insert(0, '/home/pi/GoPiGo/Software/Python/line_follower')
+        sys.path.insert(0, '/home/pi/Dexter/DI_Sensors/Python/di_sensors/red_line_follower/line_follower')
```

Comment:
if line_follower is a system-wide package, then load it via an import. 
This was a hack for when it wasn't a system-wide package.

Reply:
Okay. Then this second `try/except` block is going out, as well as the other one you mentioned.